### PR TITLE
Anna

### PR DESCRIPTION
**Pull request by AnnaLoshak**
I'm done with LogIn functiuon

### CODE REVIEW

File: src/US205/anna/LogInFunction.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package US205.anna;
+
+public class LogInFunction {
+
+    // work 1
+
+    // work 2
+
+    // work 3 is done
+}
```
Comment (BadamkhandB) [inline src/US205/anna/LogInFunction.java:13]: Looks good. keep going!

**Review by Aramat16ukr — APPROVED** [review]
good

**Review by LaylaNait — CHANGES_REQUESTED** [review]
anna add work n 4 to your work

**Review by VeronikaKrasnokutska — CHANGES_REQUESTED** [review]
add please part two under work 3

**Review by BadamkhandB — APPROVED** [review]
Perfect

**Review by VeronikaKrasnokutska — APPROVED** [review]
Looks good

**Review by VeronikaKrasnokutska — APPROVED** [review]
Good

**Comment by AnnaLoshak** [comment]
> anna add work n 4 to your work

work 4 added

**Review by Idemianiv — APPROVED** [review]
good job

**Comment by AnnaLoshak** [comment]
> anna add work n 4 to your work

work 4 added, check it please

**Comment by AnnaLoshak** [comment]
> anna add work n 4 to your work

work 4 added

File: src/US205/anna/LogInFunction.java (hunk 1 above)
Comment (natebell510) [reply src/US205/anna/LogInFunction.java:14]: Approved

**Comment by AnnaLoshak** [comment]
> anna add work n 4 to your work

work 4 added

**Review by LaylaNait — APPROVED** [review]
good

**Review by ZaierAouani — APPROVED** [review]

**Review by natebell510 — COMMENTED** [review]
Commented

**Pull request merged**